fuel-tx/src/transaction/consensus_parameters/gas/default_gas_costs.rs

```diff
@@ -161,5 +161,9 @@ pub fn default_gas_costs() -> GasCostsValues {
             base: 44,
             dep_per_unit: 5,
         },
+        contract_root: DependentCost {
+            base: 75,
+            dep_per_unit: 1,
+        },
```

Comment:
I got other results from benchmarks=) But it's okay. We can finalize them later before the final gas prices update.

<img width="206" alt="image" src="https://github.com/FuelLabs/fuel-vm/assets/18346821/e2a386ec-d7da-4e64-84d0-23abec83f0a6">